assets/pages/new-contributions-landing/components/ContributionForm.jsx

```diff
@@ -238,12 +244,18 @@ function ContributionForm(props: PropTypes) {
           <NewContributionSubmit />
           {props.isWaiting ? <ProgressMessage message={['Processing transaction', 'Please wait']} /> : null}
         </form>
+        <DirectDebitPopUpForm
+          // TODO: put payment through
+          callback={() => Promise.resolve()}
+          isPopUpOpen={props.isDirectDebitPopUpOpen}
+        />
       </div>
     );
 }
 
 ContributionForm.defaultProps = {
   error: null,
+  isDirectDebitPopUpOpen: false,
```

Comment:
not sure if I need this when the prop is linked to Redux state?

Reply:
only when it is optional, i.e. `?boolean` or `boolean | void`.